### PR TITLE
Pages

### LINKED ISSUE #24: AppOnboarding
## User story

As a **user**,
I want to have a short and simple Onboarding to the App
so I know what I can do and what I can do to use the App in an efficient way

## Notes / Links / Further Description
See design in Adobe XD

## ToDos

- [x] Program the Onboarding page
- [x] Add the routes for the links
- [x] Add the texts for Onboarding

### LINKED ISSUE #29: WelcomeScreen
## User story

As a **user** I would like to have a starting point in the app after login, from where I can navigate to the desired areas to have a better overview

## ToDos

- [x] Think about the informations that have to be shown 
- [x] Write Welcome.js
- [x] Implement in routing

### LINKED ISSUE #43: About
## User story

As a **user** I would like to know more about the app, so that I can get a better picture

## Notes / Links / Further Description
Short description of the page

## ToDos

- [x] Styling of the container
- [x] Write the text for About.js
- [x] Write About.js
- [x] Implement in Routing

### PR DESCRIPTION
**Pull request by slawoe**
Designed a few simpler pages and redesigned older ones for a better design with more whitespace, so it's a big butt.

Explanation cards are actually filled with lorem, so they look all same with different titles.

Also setup the routes for the Help pages and linked to the help-start-page for first onboarding.

Fix #24
Fix #29
Fix #43

### CODE REVIEW

File: client/src/pages/About.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import StyledTextContainer from "../components/StyledTextContainer";
+
+function About() {
+  return (
+    <>
+      <div></div>
```
Comment (manuelsanchez2) [inline client/src/pages/About.js]: What is this?

File: client/src/pages/Home.js [hunk 2]
```diff
@@ -16,12 +28,45 @@ function App() {
...
+            component={MedicsExplanation}
+          />
+          <Route
+            exact
+            path={`${path}/help/analysis`}
+            component={AnalysisExplanation}
+          />
+          <Route exact path={`${path}/about`} component={About} />
+          <Route exact path={`${path}/`} component={Welcome} />
         </Switch>
```
Comment (manuelsanchez2) [inline client/src/pages/Home.js:70]: Really cool and clean solution with the component={}

File: client/src/pages/Links.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    <>
+      <div></div>
+      <div>
+        <StyledTextContainer>
+          <h1>Nützliche Nummer und Links</h1>
+        </StyledTextContainer>
+        <StyledTextContainer>
+          <h2>Nummer gegen Kummer</h2>
+          <p>
+            <a href="https://www.nummergegenkummer.de/kinder-und-jugendtelefon.html">
```
Comment (manuelsanchez2) [inline client/src/pages/Links.js]: You can add target='_blank' so that your users never go away of the app

**Review by manuelsanchez2 — APPROVED** [review]
lgtm!

File: client/src/pages/About.js (hunk 1 above)
Comment (slawoe) [reply client/src/pages/About.js]: just for design. Will redesign it, when the rest is finished.

File: client/src/pages/Home.js (hunk 2 above)
Comment (slawoe) [reply client/src/pages/Home.js:70]: Absolutely!

File: client/src/pages/Links.js (hunk 3 above)
Comment (slawoe) [reply client/src/pages/Links.js]: will do and also insert: rel="norefferer" for security reasons

**Pull request merged**